From 511ae175e21c6b5fa32201de23b379f80ed37981 Mon Sep 17 00:00:00 2001
From: Guilherme Amadio <amadio@cern.ch>
Date: Tue, 5 Sep 2017 14:57:40 +0200
Subject: [PATCH] Do not include internal MySQL header

This breaks the build with -Dmysql=ON since the internal header will
not be found.
---
 sql/mysql/src/TMySQLServer.cxx | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/sql/mysql/src/TMySQLServer.cxx b/sql/mysql/src/TMySQLServer.cxx
index ee9d912e0f..40eebf55e9 100644
--- a/sql/mysql/src/TMySQLServer.cxx
+++ b/sql/mysql/src/TMySQLServer.cxx
@@ -52,9 +52,6 @@
 #include "TObjString.h"
 #include "TObjArray.h"
 
-#include <my_global.h>
-
-
 ClassImp(TMySQLServer)
 
 ////////////////////////////////////////////////////////////////////////////////
-- 
2.13.5

